application/backend/src/main/resources/apps/typerefinery/components/graphs/barchart/clientlibs/functions.js

```diff
@@ -0,0 +1,272 @@
+window.Typerefinery = window.Typerefinery || {};
+window.Typerefinery.Components = Typerefinery.Components || {};
+window.Typerefinery.Components.Graphs = Typerefinery.Components.Graphs || {};
+window.Typerefinery.Components.Graphs.BarChart =
+  Typerefinery.Components.Graphs.BarChart || {};
+window.Typerefinery.Components.Graphs.Items =
+  Typerefinery.Components.Graphs.Items || {};
+window.Typerefinery.Theme = Typerefinery.Theme || {};
+
+const DEFAULT_BAR_CHART_DATA = {
+  chartData: [75000, 75000, 75000, 15000, 14000, 12000, 11000, 11500, 11000],
+  labelName: "Typerefinery Bar Chart",
+  barbackgroundcolor:
+    Typerefinery?.Theme?.rootElementStyle?.getPropertyValue(
+      "--primary-object-barborder-color"
+    ) || "#FFB94E",
+  labels: [
+    "Trazen:Win32/Qakbot",
+    "Ranson:Win32/Egre..",
+    "Quakbot",
+    "Android",
+    "Emotet",
+    "XAgentOSX",
+    "Pteranodon",
+    "PsExec",
+    "Minikatz",
+  ],
+  dataSetBorderColor:
+    Typerefinery?.Theme?.rootElementStyle?.getPropertyValue(
+      "--primary-object-border-color"
+    ) || "#0099DE",
+  canvasBackgroundColor:
+    Typerefinery?.Theme?.rootElementStyle?.getPropertyValue(
+      "--subtitle-color"
```

Comment:
primary object background color should be used and "--primary-object-border-color"

Reply:
Done